### ISSUE TITLE
Istio operator fails to install demo from docs on 1.6-alpha.2 and beta.0

### ISSUE THREAD

**Issue by ericvn** [istio/istio#23236]
**Bug description**

Istio failed to be installed.

Using instructions from https://preliminary.istio.io/docs/setup/install/standalone-operator/

Running the first snippet to install demo profile.

Nothing is created in the istio-system namespace.

Logs have:
```
2020-04-24T14:41:18.010069Z	info	Detecting third-party JWT support
2020-04-24T14:41:18.488008Z	error	reconciling err: failed to create Istio control plane with spec:
profile:"demo" hub:"docker.io/istio" tag:1.6.0-alpha.2 components:<base:<enabled:<BoolValue:<value:true > > > pilot:<enabled:<BoolValue:<value:true > > tag:<nil> spec:<nil> k8s:<env:<name:"POD_NAME" valueFrom:<fieldRef:<apiVersion:"v1" fieldPath:"metadata.name" > > > env:<name:"POD_NAMESPACE" valueFrom:<fieldRef:<apiVersion:"v1" fieldPath:"metadata.namespace" > > > env:<name:"GODEBUG" value:"gctrace=1" > env:<name:"PILOT_TRACE_SAMPLING" value:"100" > env:<name:"CONFIG_NAMESPACE" value:"istio-config" > readiness_probe:<httpGet:<path:"/ready" port:8080 > initialDelaySeconds:1 timeoutSeconds:5 periodSeconds:3 > resources:<requests:<key:"cpu" value:"10m" > requests:<key:"memory" value:"100Mi" > > strategy:<rollingUpdate:<maxUnavailable:25% maxSurge:100% > > > > policy:<enabled:<BoolValue:<> > tag:<nil> spec:<nil> k8s:<env:<name:"POD_NAMESPACE" valueFrom:<fieldRef:<apiVersion:"v1" fieldPath:"metadata.namespace" > > > hpa_spec:<scaleTargetRef:<kind:"Deployment" name:"istio-policy" apiVersion:"apps/v1" > minReplicas:1 maxReplicas:5 metrics:<type:"Resource" resource:<name:"cpu" targetAverageUtilization:80 > > > resources:<requests:<key:"cpu" value:"10m" > requests:<key:"memory" value:"100Mi" > > strategy:<rollingUpdate:<maxUnavailable:25% maxSurge:100% > > > > telemetry:<enabled:<BoolValue:<> > tag:<nil> spec:<nil> k8s:<env:<name:"POD_NAMESPACE" valueFrom:<fieldRef:<apiVersion:"v1" fieldPath:"metadata.namespace" > > > env:<name:"GOMAXPROCS" value:"6" > hpa_spec:<scaleTargetRef:<kind:"Deployment" name:"istio-telemetry" apiVersion:"apps/v1" > minReplicas:1 maxReplicas:5 metrics:<type:"Resource" resource:<name:"cpu" targetAverageUtilization:80 > > > replica_count:1 resources:<limits:<key:"cpu" value:"4800m" > limits:<key:"memory" value:"4G" > requests:<key:"cpu" value:"50m" > requests:<key:"memory" value:"100Mi" > > strategy:<rollingUpdate:<maxUnavailable:25% maxSurge:100% > > > > citadel:<enabled:<BoolValue:<> > tag:<nil> spec:<nil> k8s:<strategy:<rollingUpdate:<maxUnavailable:25% maxSurge:100% > > > > cni:<enabled:<BoolValue:<> > tag:<nil> spec:<nil> > ingress_gateways:<enabled:<BoolValue:<value:true > > name:"istio-ingressgateway" tag:<nil> k8s:<resources:<requests:<key:"cpu" value:"10m" > requests:<key:"memory" value:"40Mi" > > service:<ports:<name:"status-port" port:15020 targetPort:15020 > ports:<name:"http2" port:80 targetPort:8080 > ports:<name:"https" port:443 targetPort:8443 > ports:<name:"kiali" port:15029 targetPort:15029 > ports:<name:"prometheus" port:15030 targetPort:15030 > ports:<name:"grafana" port:15031 targetPort:15031 > ports:<name:"tracing" port:15032 targetPort:15032 > ports:<name:"tcp" port:31400 targetPort:31400 > ports:<name:"tls" port:15443 targetPort:15443 > > > > egress_gateways:<enabled:<BoolValue:<value:true > > name:"istio-egressgateway" tag:<nil> k8s:<resources:<requests:<key:"cpu" value:"10m" > requests:<key:"memory" value:"40Mi" > > > > > addon_components:<key:"grafana" value:<enabled:<BoolValue:<value:true > > spec:<nil> k8s:<replica_count:1 > > > addon_components:<key:"istiocoredns" value:<enabled:<BoolValue:<> > spec:<nil> > > addon_components:<key:"kiali" value:<enabled:<BoolValue:<value:true > > spec:<nil> k8s:<replica_count:1 > > > addon_components:<key:"prometheus" value:<enabled:<BoolValue:<value:true > > spec:<nil> k8s:<replica_count:1 > > > addon_components:<key:"tracing" value:<enabled:<BoolValue:<value:true > > spec:<nil> > > values:<key:"clusterResources" value:true > values:<key:"gateways" value:map[istio-egressgateway:map[autoscaleEnabled:false name:istio-egressgateway secretVolumes:[map[mountPath:/etc/istio/egressgateway-certs name:egressgateway-certs secretName:istio-egressgateway-certs] map[mountPath:/etc/istio/egressgateway-ca-certs name:egressgateway-ca-certs secretName:istio-egressgateway-ca-certs]] type:ClusterIP] istio-ingressgateway:map[applicationPorts: autoscaleEnabled:false debug:info domain: meshExpansionPorts:[map[name:tcp-pilot-grpc-tls port:15011 targetPort:15011] map[name:tcp-istiod port:15012 targetPort:15012] map[name:tcp-citadel-grpc-tls port:8060 targetPort:8060] map[name:tcp-dns-tls port:853 targetPort:8853]] name:istio-ingressgateway secretVolumes:[map[mountPath:/etc/istio/ingressgateway-certs name:ingressgateway-certs secretName:istio-ingressgateway-certs] map[mountPath:/etc/istio/ingressgateway-ca-certs name:ingressgateway-ca-certs secretName:istio-ingressgateway-ca-certs]] type:LoadBalancer]] > values:<key:"global" value:map[arch:map[amd64:2 ppc64le:2 s390x:2] configValidation:true controlPlaneSecurityEnabled:true defaultNodeSelector:map[] defaultPodDisruptionBudget:map[enabled:true] defaultResources:map[requests:map[cpu:10m]] disablePolicyChecks:false enableHelmTest:false enableTracing:true imagePullPolicy: imagePullSecrets:[] istioNamespace:istio-system istiod:map[enableAnalysis:false enabled:true] jwtPolicy:third-party-jwt localityLbSetting:map[enabled:true] logAsJson:false logging:map[level:default:info] meshExpansion:map[enabled:false useILB:false] meshNetworks:map[] mountMtlsCerts:false mtls:map[auto:true] multiCluster:map[clusterName: enabled:false] network: omitSidecarInjectorConfigMap:false oneNamespace:false operatorManageWebhooks:false outboundTrafficPolicy:map[mode:ALLOW_ANY] pilotCertProvider:istiod policyCheckFailOpen:false priorityClassName: proxy:map[accessLogFile:/dev/stdout autoInject:enabled clusterDomain:cluster.local componentLogLevel:misc:error enableCoreDump:false envoyStatsd:map[enabled:false host:<nil> port:<nil>] excludeIPRanges: excludeInboundPorts: excludeOutboundPorts: image:proxyv2 includeIPRanges:* logLevel:warning privileged:false readinessFailureThreshold:30 readinessInitialDelaySeconds:1 readinessPeriodSeconds:2 resources:map[limits:map[cpu:2000m memory:1024Mi] requests:map[cpu:10m memory:40Mi]] statusPort:15020 tracer:zipkin] proxy_init:map[image:proxyv2 resources:map[limits:map[cpu:100m memory:50Mi] requests:map[cpu:10m memory:10Mi]]] sds:map[token:map[aud:istio-ca]] sts:map[servicePort:0] tracer:map[datadog:map[address:$(HOST_IP):8126] lightstep:map[accessToken: address: cacertPath: secure:true] stackdriver:map[debug:false maxNumberOfAnnotations:200 maxNumberOfAttributes:200 maxNumberOfMessageEvents:200] zipkin:map[address:]] trustDomain:cluster.local useMCP:false] > values:<key:"grafana" value:map[accessMode:ReadWriteMany contextPath:/grafana dashboardProviders:map[dashboardproviders.yaml:map[apiVersion:1 providers:[map[disableDeletion:false folder:istio name:istio options:map[path:/var/lib/grafana/dashboards/istio] orgId:1 type:file]]]] datasources:map[datasources.yaml:map[apiVersion:1 datasources:<nil>]] env:map[] envSecrets:map[] image:map[repository:grafana/grafana tag:6.5.2] nodeSelector:map[] persist:false podAntiAffinityLabelSelector:[] podAntiAffinityTermLabelSelector:[] security:map[enabled:false passphraseKey:passphrase secretName:grafana usernameKey:username] service:map[annotations:map[] externalPort:3000 loadBalancerIP:<nil> loadBalancerSourceRanges:<nil> name:http type:ClusterIP] storageClassName: tolerations:[]] > values:<key:"istiocoredns" value:map[coreDNSImage:coredns/coredns coreDNSPluginImage:istio/coredns-plugin:0.2-istio-1.1 coreDNSTag:1.6.2] > values:<key:"kiali" value:map[contextPath:/kiali createDemoSecret:true dashboard:map[grafanaInClusterURL:http://grafana:3000 grafanaURL:<nil> jaegerInClusterURL:http://tracing/jaeger jaegerURL:<nil> passphraseKey:passphrase secretName:kiali usernameKey:username viewOnlyMode:false] hub:quay.io/kiali nodeSelector:map[] podAntiAffinityLabelSelector:[] podAntiAffinityTermLabelSelector:[] prometheusNamespace:<nil> security:map[cert_file:/kiali-cert/cert-chain.pem enabled:false private_key_file:/kiali-cert/key.pem] tag:v1.15] > values:<key:"meshConfig" value:map[] > values:<key:"mixer" value:map[adapters:map[kubernetesenv:map[enabled:true] prometheus:map[enabled:true metricsExpiryDuration:10m] stackdriver:map[auth:map[apiKey: appCredentials:false serviceAccountPath:] enabled:false tracer:map[enabled:false sampleProbability:1]] stdio:map[enabled:true outputAsJson:false] useAdapterCRDs:false] policy:map[adapters:map[kubernetesenv:map[enabled:true] useAdapterCRDs:false] autoscaleEnabled:false image:mixer sessionAffinityEnabled:false] telemetry:map[autoscaleEnabled:false env:map[GOMAXPROCS:6] image:mixer loadshedding:map[latencyThreshold:100ms mode:enforce] nodeSelector:map[] podAntiAffinityLabelSelector:[] podAntiAffinityTermLabelSelector:[] replicaCount:1 sessionAffinityEnabled:false tolerations:[]]] > values:<key:"pilot" value:map[appNamespaces:[] autoscaleEnabled:false autoscaleMax:5 autoscaleMin:1 configMap:true configNamespace:istio-config cpu:map[targetAverageUtilization:80] deploymentLabels:<nil> enableProtocolSniffingForInbound:true enableProtocolSniffingForOutbound:true env:map[] image:pilot keepaliveMaxServerConnectionAge:30m nodeSelector:map[] podAntiAffinityLabelSelector:[] podAntiAffinityTermLabelSelector:[] policy:map[enabled:false] replicaCount:1 tolerations:[] traceSampling:1] > values:<key:"prometheus" value:map[contextPath:/prometheus hub:docker.io/prom nodeSelector:map[] podAntiAffinityLabelSelector:[] podAntiAffinityTermLabelSelector:[] provisionPrometheusCert:true retention:6h scrapeInterval:15s security:map[enabled:true] tag:v2.15.1 tolerations:[]] > values:<key:"sidecarInjectorWebhook" value:map[enableNamespacesByDefault:false injectLabel:istio-injection objectSelector:map[autoInject:true enabled:false] rewriteAppHTTPProbe:true] > values:<key:"telemetry" value:map[enabled:true v1:map[enabled:false] v2:map[enabled:true prometheus:map[enabled:true] stackdriver:map[configOverride:map[] enabled:false logging:false monitoring:false topology:false]]] > values:<key:"tracing" value:map[jaeger:map[accessMode:ReadWriteMany hub:docker.io/jaegertracing memory:map[max_traces:50000] persist:false spanStorageType:badger storageClassName: tag:1.16] nodeSelector:map[] opencensus:map[exporters:map[stackdriver:map[enable_tracing:true]] hub:docker.io/omnition resources:map[limits:map[cpu:1 memory:2Gi] requests:map[cpu:200m memory:400Mi]] tag:0.1.9] podAntiAffinityLabelSelector:[] podAntiAffinityTermLabelSelector:[] provider:jaeger service:map[annotations:map[] externalPort:9411 name:http-query type:ClusterIP] zipkin:map[hub:docker.io/openzipkin javaOptsHeap:700 maxSpans:500000 node:map[cpus:2] probeStartupDelay:10 queryPort:9411 resources:map[limits:map[cpu:1000m memory:2048Mi] requests:map[cpu:150m memory:900Mi]] tag:2.20.0]] > values:<key:"version" value: >
error: compiled in charts not found in this development build, use --charts with local charts instead or run make gen-charts
2020-04-24T14:41:19.519431Z	info	Reconciling IstioOperator
2020-04-24T14:41:19.519847Z	info	Updating IstioOperator
2020-04-24T14:41:19.519871Z	info	installer	ReadProfileYAML for profile name: demo
2020-04-24T14:41:19.519881Z	info	installer	Loading values from compiled in VFS at path profiles/demo.yaml
2020-04-24T14:41:19.519892Z	info	installer	ReadProfileYAML for profile name: default
2020-04-24T14:41:19.519898Z	info	installer	Loading values from compiled in VFS at path profiles/default.yaml
```

**Expected behavior**

**Steps to reproduce the bug**

**Version (include the output of `istioctl version --remote` and `kubectl version` and `helm version` if you used Helm)**

**How was Istio installed?**

**Environment where bug was observed (cloud vendor, OS, etc)**

**Comment by ericvn** [comment]
I tried again with `istio-1.6.0-beta.0`. Still fails to deploy.

Logs:
```
2020-04-28T14:28:37.933005Z	info	ControlZ available at 127.0.0.1:9876
2020-04-28T14:28:38.394099Z	info	Registering Components.
2020-04-28T14:28:38.394314Z	info	Adding controller for IstioOperator
2020-04-28T14:28:38.394461Z	info	Controller added
2020-04-28T14:28:38.394506Z	info	Starting the Cmd.
2020-04-28T14:28:38.394722Z	info	attempting to acquire leader lease  istio-operator/istio-operator-lock...
2020-04-28T14:28:38.426736Z	info	successfully acquired lease istio-operator/istio-operator-lock
2020-04-28T14:28:47.298851Z	info	Reconciling IstioOperator
2020-04-28T14:28:47.299244Z	info	Adding finalizer istio-finalizer.install.istio.io to istio-system/example-istiocontrolplane
2020-04-28T14:28:47.316226Z	info	Updating IstioOperator
2020-04-28T14:28:47.316310Z	info	installer	ReadProfileYAML for profile name: demo
2020-04-28T14:28:47.316321Z	info	installer	Loading values from compiled in VFS at path profiles/demo.yaml
2020-04-28T14:28:47.316332Z	info	installer	ReadProfileYAML for profile name: default
2020-04-28T14:28:47.316337Z	info	installer	Loading values from compiled in VFS at path profiles/default.yaml
2020-04-28T14:28:47.393805Z	info	proto: tag has too few fields: "-"
2020-04-28T14:28:47.894105Z	info	Detecting third-party JWT support
2020-04-28T14:28:48.394652Z	error	reconciling err: failed to create Istio control plane with spec:
profile:"demo" hub:"docker.io/istio" tag:1.6.0-beta.0 components:<base:<enabled:<BoolValue:<value:true > > > pilot:<enabled:<BoolValue:<value:true > > tag:<nil> spec:<nil> k8s:<env:<name:"POD_NAME" valueFrom:<fieldRef:<apiVersion:"v1" fieldPath:"metadata.name" > > > env:<name:"POD_NAMESPACE" valueFrom:<fieldRef:<apiVersion:"v1" fieldPath:"metadata.namespace" > > > env:<name:"GODEBUG" value:"gctrace=1" > env:<name:"PILOT_TRACE_SAMPLING" value:"100" > env:<name:"CONFIG_NAMESPACE" value:"istio-config" > readiness_probe:<httpGet:<path:"/ready" port:8080 > initialDelaySeconds:1 timeoutSeconds:5 periodSeconds:3 > resources:<requests:<key:"cpu" value:"10m" > requests:<key:"memory" value:"100Mi" > > strategy:<rollingUpdate:<maxUnavailable:25% maxSurge:100% > > > > policy:<enabled:<BoolValue:<> > tag:<nil> spec:<nil> k8s:<env:<name:"POD_NAMESPACE" valueFrom:<fieldRef:<apiVersion:"v1" fieldPath:"metadata.namespace" > > > hpa_spec:<scaleTargetRef:<kind:"Deployment" name:"istio-policy" apiVersion:"apps/v1" > minReplicas:1 maxReplicas:5 metrics:<type:"Resource" resource:<name:"cpu" targetAverageUtilization:80 > > > resources:<requests:<key:"cpu" value:"10m" > requests:<key:"memory" value:"100Mi" > > strategy:<rollingUpdate:<maxUnavailable:25% maxSurge:100% > > > > telemetry:<enabled:<BoolValue:<> > tag:<nil> spec:<nil> k8s:<env:<name:"POD_NAMESPACE" valueFrom:<fieldRef:<apiVersion:"v1" fieldPath:"metadata.namespace" > > > env:<name:"GOMAXPROCS" value:"6" > hpa_spec:<scaleTargetRef:<kind:"Deployment" name:"istio-telemetry" apiVersion:"apps/v1" > minReplicas:1 maxReplicas:5 metrics:<type:"Resource" resource:<name:"cpu" targetAverageUtilization:80 > > > replica_count:1 resources:<limits:<key:"cpu" value:"4800m" > limits:<key:"memory" value:"4G" > requests:<key:"cpu" value:"50m" > requests:<key:"memory" value:"100Mi" > > strategy:<rollingUpdate:<maxUnavailable:25% maxSurge:100% > > > > citadel:<enabled:<BoolValue:<> > tag:<nil> spec:<nil> k8s:<strategy:<rollingUpdate:<maxUnavailable:25% maxSurge:100% > > > > cni:<enabled:<BoolValue:<> > tag:<nil> spec:<nil> > ingress_gateways:<enabled:<BoolValue:<value:true > > name:"istio-ingressgateway" tag:<nil> k8s:<resources:<requests:<key:"cpu" value:"10m" > requests:<key:"memory" value:"40Mi" > > service:<ports:<name:"status-port" port:15020 targetPort:15020 > ports:<name:"http2" port:80 targetPort:8080 > ports:<name:"https" port:443 targetPort:8443 > ports:<name:"kiali" port:15029 targetPort:15029 > ports:<name:"prometheus" port:15030 targetPort:15030 > ports:<name:"grafana" port:15031 targetPort:15031 > ports:<name:"tracing" port:15032 targetPort:15032 > ports:<name:"tcp" port:31400 targetPort:31400 > ports:<name:"tls" port:15443 targetPort:15443 > > > > egress_gateways:<enabled:<BoolValue:<value:true > > name:"istio-egressgateway" tag:<nil> k8s:<resources:<requests:<key:"cpu" value:"10m" > requests:<key:"memory" value:"40Mi" > > > > > addon_components:<key:"grafana" value:<enabled:<BoolValue:<value:true > > spec:<nil> k8s:<replica_count:1 > > > addon_components:<key:"istiocoredns" value:<enabled:<BoolValue:<> > spec:<nil> > > addon_components:<key:"kiali" value:<enabled:<BoolValue:<value:true > > spec:<nil> k8s:<replica_count:1 > > > addon_components:<key:"prometheus" value:<enabled:<BoolValue:<value:true > > spec:<nil> k8s:<replica_count:1 > > > addon_components:<key:"tracing" value:<enabled:<BoolValue:<value:true > > spec:<nil> > > values:<key:"clusterResources" value:true > values:<key:"gateways" value:map[istio-egressgateway:map[autoscaleEnabled:false name:istio-egressgateway secretVolumes:[map[mountPath:/etc/istio/egressgateway-certs name:egressgateway-certs secretName:istio-egressgateway-certs] map[mountPath:/etc/istio/egressgateway-ca-certs name:egressgateway-ca-certs secretName:istio-egressgateway-ca-certs]] type:ClusterIP] istio-ingressgateway:map[applicationPorts: autoscaleEnabled:false debug:info domain: meshExpansionPorts:[map[name:tcp-pilot-grpc-tls port:15011 targetPort:15011] map[name:tcp-istiod port:15012 targetPort:15012] map[name:tcp-citadel-grpc-tls port:8060 targetPort:8060] map[name:tcp-dns-tls port:853 targetPort:8853]] name:istio-ingressgateway secretVolumes:[map[mountPath:/etc/istio/ingressgateway-certs name:ingressgateway-certs secretName:istio-ingressgateway-certs] map[mountPath:/etc/istio/ingressgateway-ca-certs name:ingressgateway-ca-certs secretName:istio-ingressgateway-ca-certs]] type:LoadBalancer]] > values:<key:"global" value:map[arch:map[amd64:2 ppc64le:2 s390x:2] configValidation:true controlPlaneSecurityEnabled:true defaultNodeSelector:map[] defaultPodDisruptionBudget:map[enabled:true] defaultResources:map[requests:map[cpu:10m]] disablePolicyChecks:false enableHelmTest:false enableTracing:true imagePullPolicy: imagePullSecrets:[] istioNamespace:istio-system istiod:map[enableAnalysis:false enabled:true] jwtPolicy:third-party-jwt localityLbSetting:map[enabled:true] logAsJson:false logging:map[level:default:info] meshExpansion:map[enabled:false useILB:false] meshNetworks:map[] mountMtlsCerts:false mtls:map[auto:true] multiCluster:map[clusterName: enabled:false] network: omitSidecarInjectorConfigMap:false oneNamespace:false operatorManageWebhooks:false outboundTrafficPolicy:map[mode:ALLOW_ANY] pilotCertProvider:istiod policyCheckFailOpen:false priorityClassName: proxy:map[accessLogFile:/dev/stdout autoInject:enabled clusterDomain:cluster.local componentLogLevel:misc:error enableCoreDump:false envoyStatsd:map[enabled:false host:<nil> port:<nil>] excludeIPRanges: excludeInboundPorts: excludeOutboundPorts: image:proxyv2 includeIPRanges:* logLevel:warning privileged:false readinessFailureThreshold:30 readinessInitialDelaySeconds:1 readinessPeriodSeconds:2 resources:map[limits:map[cpu:2000m memory:1024Mi] requests:map[cpu:10m memory:40Mi]] statusPort:15020 tracer:zipkin] proxy_init:map[image:proxyv2 resources:map[limits:map[cpu:100m memory:50Mi] requests:map[cpu:10m memory:10Mi]]] sds:map[token:map[aud:istio-ca]] sts:map[servicePort:0] tracer:map[datadog:map[address:$(HOST_IP):8126] lightstep:map[accessToken: address: cacertPath: secure:true] stackdriver:map[debug:false maxNumberOfAnnotations:200 maxNumberOfAttributes:200 maxNumberOfMessageEvents:200] zipkin:map[address:]] trustDomain:cluster.local useMCP:false] > values:<key:"grafana" value:map[accessMode:ReadWriteMany contextPath:/grafana dashboardProviders:map[dashboardproviders.yaml:map[apiVersion:1 providers:[map[disableDeletion:false folder:istio name:istio options:map[path:/var/lib/grafana/dashboards/istio] orgId:1 type:file]]]] datasources:map[datasources.yaml:map[apiVersion:1 datasources:<nil>]] env:map[] envSecrets:map[] image:map[repository:grafana/grafana tag:6.5.2] nodeSelector:map[] persist:false podAntiAffinityLabelSelector:[] podAntiAffinityTermLabelSelector:[] security:map[enabled:false passphraseKey:passphrase secretName:grafana usernameKey:username] service:map[annotations:map[] externalPort:3000 loadBalancerIP:<nil> loadBalancerSourceRanges:<nil> name:http type:ClusterIP] storageClassName: tolerations:[]] > values:<key:"istiocoredns" value:map[coreDNSImage:coredns/coredns coreDNSPluginImage:istio/coredns-plugin:0.2-istio-1.1 coreDNSTag:1.6.2] > values:<key:"kiali" value:map[contextPath:/kiali createDemoSecret:true dashboard:map[grafanaInClusterURL:http://grafana:3000 grafanaURL:<nil> jaegerInClusterURL:http://tracing/jaeger jaegerURL:<nil> passphraseKey:passphrase secretName:kiali usernameKey:username viewOnlyMode:false] hub:quay.io/kiali nodeSelector:map[] podAntiAffinityLabelSelector:[] podAntiAffinityTermLabelSelector:[] prometheusNamespace:<nil> security:map[cert_file:/kiali-cert/cert-chain.pem enabled:false private_key_file:/kiali-cert/key.pem] tag:v1.15] > values:<key:"meshConfig" value:map[] > values:<key:"mixer" value:map[adapters:map[kubernetesenv:map[enabled:true] prometheus:map[enabled:true metricsExpiryDuration:10m] stackdriver:map[auth:map[apiKey: appCredentials:false serviceAccountPath:] enabled:false tracer:map[enabled:false sampleProbability:1]] stdio:map[enabled:true outputAsJson:false] useAdapterCRDs:false] policy:map[adapters:map[kubernetesenv:map[enabled:true] useAdapterCRDs:false] autoscaleEnabled:false image:mixer sessionAffinityEnabled:false] telemetry:map[autoscaleEnabled:false env:map[GOMAXPROCS:6] image:mixer loadshedding:map[latencyThreshold:100ms mode:enforce] nodeSelector:map[] podAntiAffinityLabelSelector:[] podAntiAffinityTermLabelSelector:[] replicaCount:1 sessionAffinityEnabled:false tolerations:[]]] > values:<key:"pilot" value:map[appNamespaces:[] autoscaleEnabled:false autoscaleMax:5 autoscaleMin:1 configMap:true configNamespace:istio-config cpu:map[targetAverageUtilization:80] deploymentLabels:<nil> enableProtocolSniffingForInbound:true enableProtocolSniffingForOutbound:true env:map[] image:pilot keepaliveMaxServerConnectionAge:30m nodeSelector:map[] podAntiAffinityLabelSelector:[] podAntiAffinityTermLabelSelector:[] policy:map[enabled:false] replicaCount:1 tolerations:[] traceSampling:1] > values:<key:"prometheus" value:map[contextPath:/prometheus hub:docker.io/prom nodeSelector:map[] podAntiAffinityLabelSelector:[] podAntiAffinityTermLabelSelector:[] provisionPrometheusCert:true retention:6h scrapeInterval:15s security:map[enabled:true] tag:v2.15.1 tolerations:[]] > values:<key:"sidecarInjectorWebhook" value:map[enableNamespacesByDefault:false injectLabel:istio-injection objectSelector:map[autoInject:true enabled:false] rewriteAppHTTPProbe:true] > values:<key:"telemetry" value:map[enabled:true v1:map[enabled:false] v2:map[enabled:true prometheus:map[enabled:true] stackdriver:map[configOverride:map[] enabled:false logging:false monitoring:false topology:false]]] > values:<key:"tracing" value:map[jaeger:map[accessMode:ReadWriteMany hub:docker.io/jaegertracing memory:map[max_traces:50000] persist:false spanStorageType:badger storageClassName: tag:1.16] nodeSelector:map[] opencensus:map[exporters:map[stackdriver:map[enable_tracing:true]] hub:docker.io/omnition resources:map[limits:map[cpu:1 memory:2Gi] requests:map[cpu:200m memory:400Mi]] tag:0.1.9] podAntiAffinityLabelSelector:[] podAntiAffinityTermLabelSelector:[] provider:jaeger service:map[annotations:map[] externalPort:9411 name:http-query type:ClusterIP] zipkin:map[hub:docker.io/openzipkin javaOptsHeap:700 maxSpans:500000 node:map[cpus:2] probeStartupDelay:10 queryPort:9411 resources:map[limits:map[cpu:1000m memory:2048Mi] requests:map[cpu:150m memory:900Mi]] tag:2.20.0]] > values:<key:"version" value: >
error: compiled in charts not found in this development build, use --charts with local charts instead or run make gen-charts
2020-04-28T14:28:48.412045Z	info	Reconciling IstioOperator
2020-04-28T14:28:48.412188Z	info	Updating IstioOperator
2020-04-28T14:28:48.412199Z	info	installer	ReadProfileYAML for profile name: demo
2020-04-28T14:28:48.412218Z	info	installer	Loading values from compiled in VFS at path profiles/demo.yaml
2020-04-28T14:28:48.412229Z	info	installer	ReadProfileYAML for profile name: default
2020-04-28T14:28:48.412235Z	info	installer	Loading values from compiled in VFS at path profiles/default.yaml
2020-04-28T14:28:49.014577Z	info	Detecting third-party JWT support
...
```

**Comment by ericvn** [comment]
Still fails on 1.6.0-beta.1

**Comment by ostromart** [comment]
Should be fixed in https://github.com/istio/release-builder/pull/243.

**Comment by ericvn** [comment]
Fixed in 1.6.0-rc.o
